src/rtk/features/singleRights/singleRightsSlice.ts

```diff
@@ -190,19 +205,24 @@ const singleRightSlice = createSlice({
         }
       })
       .addCase(delegate.fulfilled, (state, action) => {
-        const delegationInput = createSerializedDelegationInput(
+        const delegationInput = createSerializedMeta(
           action.meta.arg,
           ReduxStatusResponse.Fulfilled,
         );
-        state.processedDelegations.push(delegationInput);
+
+        const pushData: ProcessedDelegation = {
+          meta: delegationInput,
+          bffResponseList: action.payload.rightDelegationResults,
+        };
+        state.processedDelegations.push(pushData);
       })
       .addCase(delegate.rejected, (state, action) => {
-        console.log('addCaseDelegateAction', action);
-        const delegationInput = createSerializedDelegationInput(
-          action.meta.arg,
-          ReduxStatusResponse.Rejected,
-        );
-        state.processedDelegations.push(delegationInput);
+        const delegationInput = createSerializedMeta(action.meta.arg, ReduxStatusResponse.Rejected);
+
+        const pushData: ProcessedDelegation = {
```

Comment:
I only added support in the BFF for responses on Det magiske klesskapet, Brannbilen i Brønnøysund, Snarveien - Lenketjeneste and A3 app 2 til å delegere. Probably should have stated it in this PR description. Responses like this is really just because the response from the BFF returns no data when we don't have responses for them setup and I think it's redundant to do anything about it before we know how the backend will respond.

Think it's smarter to do this in the task where we integrate the BFF with backend, because then we will know for sure how all the different responses will be.

Reply:
Then I think we should do it in it's own task as we need to add logic to reject cases that isn't found in the BFF and also process all the different errors and handle all those errors. The task also probably needs some grooming and additional design for showing errors like that.

Reply:
I'll check it out a bit on why the responses aren't rejected like they should but I think there was a problem where Redux couldn't deserialize the ProblemDetails from BFF which was too big to fix in this task and I also would have needed all clearified what's mentioned above in order to do this properly.

Reply:
Also since BFF can't return 400 and 500-errors organically before backend is done, I think it is in fact in the scope of an own task or in the integrate with backend-task as it isn't in either the description or the AC of the task.

Reply:
Nvm, wasn't so complicated to implment this and I just used the design we previously was gonna use when all action delegations fails

Reply:
Awesome! 😄